aggregator/src/aggregator/http_handlers.rs

```diff
@@ -266,6 +266,34 @@ where
     }
 }
 
+/// A Trillium handler that returns an empty body.
+#[derive(Clone, Copy)]
+struct EmptyBody {
+    media_type: &'static str,
+    retry_after: &'static str, // TODO: Make this editable? TKTK
+    status: Status,            // TODO: Make this editable? TKTK
```

Comment:
I have two lines marked `TODO` in here that I'd like some feedback on: They are the `Status` and `Retry-After` fields returned with an empty body in replacement of the `Processing` enumeration.

Right now I'm hard-coding these to 2 seconds and 200 OK, so there's no reason for them to be called out here into the struct. 

As far as I can tell this is fine for the CollectionJobResp as well, so it seems to me that Status should be just hardcoded, problem solved.

But what about the Retry-After? Should we make that variable at all here, now, or should I simplify this struct because YAGNI?

Reply:
I think we could just hardcode 200 and it'd be fine. Unless I made a mistake when writing the spec (totally possible!) then you shouldn't ever need a more specific HTTP status to make this unambiguous.

I don't think we're particularly clever about the retry-after. I'm not sure what we would use to estimate this, is the thing. I think a reasonable first try would be to use the interval at which the aggregation job driver polls for runnable aggregation jobs, and maybe throw 20-30% on top of that to account for the processing time.